src/main/java/com/microsoft/sqlserver/jdbc/SQLServerADAL4JUtils.java

```diff
@@ -28,7 +28,7 @@
     static final private java.util.logging.Logger adal4jLogger = java.util.logging.Logger
             .getLogger("com.microsoft.sqlserver.jdbc.internals.SQLServerADAL4JUtils");
 
-    static SqlFedAuthToken getSqlFedAuthToken(SqlFedAuthInfo fedAuthInfo, String user, String password,
+    static String getSqlFedAuthToken(SqlFedAuthInfo fedAuthInfo, String user, String password,
```

Comment:
I see that you've changed all of our current methods that return a SqlFedAuthToken to return string (the string representing the access token). I understand that MSI access tokens last 8 hours instead of the usual 1 hour ones from Azure, but we use these methods for getting the access tokens from Azure as well. If we want to add token refresh logic to our driver in the future, we're going to have to revert these methods so that we retrieve both the refresh token and access token (and store it as a SqlFedAuthToken, but this PR removes that class). Did I miss something regarding how our driver will handle refresh tokens in the future, or have you already considered this case?